langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -74,8 +132,22 @@ public isolated function round(decimal x, int fractionDigits = 0) returns decima
     name: "round"
 } external;
 
-# IEEE quantize operation.
-
+# Return a decimal with a specified value and exponent.
+# Return a decimal value that has the same value (except for rounding) as the first
+# argument, and the same exponent as the second argument.
+# This is the IEEE quantize operation.
+#
+# ```ballerina
+# decimal d = 4.1626;
+# d.quantize(3.512) ⇒ 4.163
+# 
+# decimal:quantize(4.1626, 3.512) ⇒ 4.163
+# 
+# decimal:quantize(4.1624, 3.512) ⇒ 4.162
```

Comment:
Not really needed, right?

Reply:
it added to show the `round` property. will remove it